docs/stable/1-client.md

```diff
@@ -94,33 +96,39 @@ To communicate between Status nodes, the [RLPx Transport
 Protocol, v5](https://github.com/ethereum/devp2p/blob/master/rlpx.md) is used, which
 allows for TCP-based communication between nodes.
 
-On top of this we run the RLPx-based subprotocol [Whisper
-v6](https://eips.ethereum.org/EIPS/eip-627) for privacy-preserving messaging.
+On top of this we run RLPx-based subprotocols, the client
+MAY use [Whisper v6](https://eips.ethereum.org/EIPS/eip-627), though the client 
+SHOULD use [Waku V1](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-1.md)
+for privacy-preserving messaging.
 
-There MUST be a node that is capable of discovering peers and
-implements Whisper V6 specification.
+There MUST be a node that is capable of discovering peers and implements
+[Whisper v6](https://eips.ethereum.org/EIPS/eip-627),
+[Waku V0](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-0.md) and
+[Waku V1](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-1.md) specifications.
 
 #### Node discovery and roles
 
-There are four types of node roles:
-1. Bootstrap nodes
-2. Whisper relayers
-3. Mailservers (servers and clients)
-4. Mobile nodes (Status Clients)
+There are five types of node roles:
+1. `Bootstrap node`
+1. `Waku relayer`
+1. `Whisper relayer`
+1. `Mailserver` (servers and clients)
+1. `Mobile node` (Status Clients)
 
-To implement a standard Status client you MUST implement both 2. and 4. node types. The
-other node types are optional, but we RECOMMEND you implement a mailserver
-client mode, otherwise the user experience is likely to be poor.
+To implement a standard Status client you MUST implement `Waku relayer` and/or `Whisper relayer` roles,
```

Comment:
```suggestion
A standard Status client MUST implement `Waku relayer` and/or `Whisper relayer` roles,
```

I don't like the usage of you

Reply:
Neither do I.  There's quite a bit of the use of personal pronouns in these specs. I'll keep an eye out for them and replace them as I come across them.